src/gapstate.c

```diff
@@ -5,30 +5,91 @@
  **
  ** This file declares all variables that are considered state for the
  ** interpreter
```

Comment:
The above comment (which isn't part of this PR, of course) is actually wrong, or at best, misleading...